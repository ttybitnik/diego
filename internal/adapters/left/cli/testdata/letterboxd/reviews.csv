Date,Name,Year,Letterboxd URI,Rating,Rewatch,Review,Tags,Watched Date