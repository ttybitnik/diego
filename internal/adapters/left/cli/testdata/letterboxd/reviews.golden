- name: Fallen Leaves
  year: "2023"
  url: https://boxd.it/5wluaZ
  date: "2024-01-07"
  rating: "4"
  review: Lorem Ipsum!
- name: Goodbye, Dragon Inn
  year: "2003"
  url: https://boxd.it/5E4WzB
  date: "2024-01-21"
  rating: "4"
  review: Lorem Ipsum!
- name: Pictures of Ghosts
  year: "2023"
  url: https://boxd.it/5FB5dd
  date: "2024-01-24"
  rating: "5"
  review: Lorem Ipsum!
